my-app/src/Components/Reviews/ListReviews.jsx

```diff
@@ -90,97 +109,104 @@ function ListReviews() {
+              <h3 className="review-notify">Waiting for loading data!</h3>
             )}
-          {JSON.stringify(dataPerPage) !== JSON.stringify([]) && (
-            <div>
-              <table className="table table-bordered text-center">
-                <thead>
-                  <tr>
-                    <th>#</th>
-                    <th>User is checked</th>
-                    <th className="role-list">Role</th>
-                    <th className="view-review">Check</th>
-                  </tr>
-                </thead>
-                <tbody>
-                  {dataPerPage?.map((ele, index) => {
-                    return (
-                      <tr key={index}>
-                        <td>{(page - 1) * itemsPerPage + index + 1}</td>
-                        <td>
-                          <a style={{ textDecoration: "none" }}>
-                            {ele.user.first_name !== null &&
-                            ele.user.first_name !== null
-                              ? ele.user.first_name +
-                                " " +
-                                ele.user.last_name +
-                                " (" +
-                                ele.user.email +
-                                " )"
-                              : ele.user.email}
-                          </a>
-                        </td>
-                        <td>
-                          <select
-                            className="form-select list-reviews"
-                            aria-label="Default select example"
-                            value={ele.user.role_id}
-                            id={ele.id}
-                            disabled
-                          >
-                            <option value="1">Group leader</option>
-                            <option value="2">Leader</option>
-                            <option value="3">Member</option>
-                          </select>
-                        </td>
-                        <td>
-                          <a
-                            href={`/mycheckpoints/${params.check_id}/reviews/${
-                              ele.id
-                            }?title=${title}&user_id=${ele.user.id}&username=${
-                              ele.user.first_name !== null &&
-                              ele.user.first_name !== null
-                                ? ele.user.first_name +
-                                  " " +
-                                  ele.user.last_name +
-                                  " (" +
-                                  ele.user.email +
-                                  " )"
-                                : ele.user.email
-                            }`}
-                          >
-                            <button
-                              variant="primary"
-                              className="btn-list-review"
+            {JSON.stringify(dataPerPage) === JSON.stringify([]) &&
```

Comment:
mục đích check như vậy để làm gì em

Reply:
em thấy khi fetch Data thì có 1 khoảng tgian đợi, để phân biệt 3TH để hiển thị thông báo lên màn hình cho client:
1.  lúc đang tải dữ liệu -> waiting for loading data
2.  lúc tải xong mà ko có dữ liệu -> no data exists
3.  lúc tải xong có dữ liệu -> hiển thị dữ liệu

Reply:
```dataPerPage``` là 1 array đúng ko, nếu đúng thì em có thể check length của nó mà

Reply:
trường hợp em chỉ cần check array có rỗng hay không thì chỉ cần check length thôi

Reply:
ok anh, em hiểu rồi anh